classes/Domain/Entity/Mapper/Talk.php

```diff
@@ -23,10 +23,10 @@ class Talk extends Mapper
      * user has favourited this talk
      *
      * @param integer $admin_user_id
-     * @param string $order_by
-     * @param string $sort Sort Direction
-     * @throws InvalidArgumentException If order by is not in white list
-     * @return array
+     * @param array $options
+     * @return array If order by is not in white list
+     * @internal param string $order_by
```

Comment:
Yeah, it gets over-zealously helpful sometimes. I just thought it was funny that as soon as I saw that, I knew you were using PhpStorm hahaha.